modules/codegen/src/main/scala/com/twilio/guardrail/SwaggerUtil.scala

```diff
@@ -141,11 +143,13 @@ object SwaggerUtil {
     } yield CustomTypeName(v, prefixes)
 
   sealed class ModelMetaTypePartiallyApplied[L <: LA, F[_]](val dummy: Boolean = true) {
-    def apply[T <: Schema[_]](model: T)(implicit Sc: ScalaTerms[L, F], Sw: SwaggerTerms[L, F], Fw: FrameworkTerms[L, F]): Free[F, ResolvedType[L]] =
+    def apply[T <: Schema[_]](
+        model: T
+    )(implicit Sc: ScalaTerms[L, F], Sw: SwaggerTerms[L, F], F: FrameworkTerms[L, F]): Free[F, ResolvedType[L]] =
       Sw.log.function("modelMetaType") {
         import Sc._
         import Sw._
-        import Fw._
+        import F._
```

Comment:
Minor nit: I kinda prefer this as `Fw` so it doesn't get confused with the type param `F`.

Reply:
Yep, not sure why this got changed, but changing it back now.